docs/how-to-guides/feathr-spark-udf-advanced.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: default
+title: Developing Customized Feathr Spark UDF
+parent: How-to Guides
+---
+
+# Developing Customized Feathr Spark UDF
+
+Feathr provides flexible ways for end users to define featurization logics. One of the advanced use case would be to use complex, customized logic to do transformations in Feathr. This document describes the required steps to do that.
+
+Although Feathr uses Spark as execution engine, most of the users don't have to know that; However for advanced use cases such as using Spark UDF, users have to know a bit on Spark.
```

Comment:
- as `the` execution engine.
- "it could be transparent to most of the users" would be better?
- "users need to have a basic knowledge of Spark" ?